### PR TITLE
Fix enhancement list

### PR DESCRIPTION
**Pull request by IssaicBelden**
CRUD is now functional. Needs Further enhancement on new branch.

### CODE REVIEW

File: src/TaskIt.Controllers/TasksController.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        public ActionResult Index()
+        {
+            return View(Context.Tasks);
+
+        }
+
+        public ActionResult Details(int id)
+        {
+            var tasks = Context.Tasks.
+                Where(t => t.TaskId == id);
```
Comment (JoshuaBelden) [inline src/TaskIt.Controllers/TasksController.cs]: Usually you want to drop a long line down at the '.'
For instance
```
var tasks = Context
   .Tasks
   .Where(t => t.TaskId == id);
```

File: src/TaskIt.Controllers/TasksController.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        {
+            Task task = new Task
+            {
+                Title = tvalue,
+                Description = dvalue
+            };
+            if(task.Title != null)
+            {
+            Context.Tasks.Add(task);
+            Context.SaveChanges();
```
Comment (JoshuaBelden) [inline src/TaskIt.Controllers/TasksController.cs]: Make sure you're indenting with in braces.

File: src/TaskIt.Controllers/TasksController.cs [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+            }
+            return View();
+        }
+
+        public ActionResult Remove(int id)
+        {
+            Context.Tasks.RemoveRange(Context.Tasks.Where(t => t.TaskId == id));
```
Comment (JoshuaBelden) [inline src/TaskIt.Controllers/TasksController.cs]: Ideally, you'd never have more than 1 task with a particular id. You can use Tasks.Remove and hand it a single task. The best way to get a single task is with the First method.

```
Context.Tasks.Remove(Context.Tasks.First(t => t.TaskId == id));
```

**Comment by IssaicBelden** [comment]
Made changes to new branch and will commit that one. Solved button issue faster than expected.

**Pull request closed without merging**